test/fixtures/es-module-loaders/reserved-keywords.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  enum: 'enum',
+  class: 'class',
+  delete: 'delete',
+  [Symbol.for('esModuleInterop')]: true,
```

Comment:
Babel folks any opinions on the symbol name `esModuleInterop`?
\@hzoo @danez @loganfsmyth

Reply:
there's no need to force camelCase. `ES Module Interop` would work too.

Reply:
most symbols i see throughout the js world use camelCase or PascalCase

Reply:
Variable names do; Symbols haven't been around long enough to have an established convention around that. The string in `Symbol()` and `Symbol.for()` is a *description* - there's zero reason it needs to be limited to being an identifier.

Reply:
I did not see the camel case debate coming... was more on whether it should be `Symbol.for('esModule')` so just remove the double underbar but sure. Node is currently a bit all over the place with its symbol labels. Examples:
```
Symbol('util.promisify.custom')
Symbol('customPromisifyArgs')
```

I was also curious if Babel was transitioning to a symbol key and if they'd already named it.

Reply:
I also wanted to pull them (indirectly) into this thread since its super relevant to them.

Reply:
Definitely appreciate the ping. I personally don't see that it'd be feasible for us to use a Symbol because the code could be running on a platform that doesn't support Symbols, assuming I'm following all this properly, but it probably does make sense to use a Symbol in the long run for loaders, assuming Symbol support is known to exist.